spec/index.bs

```diff
@@ -8612,6 +8612,10 @@ will affect a render pass's {{GPURenderPassDescriptor/depthStencilAttachment}}:
     - If |descriptor|.{{GPUDepthStencilState/stencilFront}} or
```

Comment:
The above phrasing states that if `depthCompare != "always"`, then `format` must have a depth component.

Now that `depthCompare` is optional, should that be interpreted as if `depthCompare != undefined && depthCompare != "always"`, then `format` must have a depth component?

I.e. what should happen if I don't fill in `depthCompare`, or I fill in `.depthCompare = undefined;`?

Reply:
(Thread moved to #4508)